homeassistant/components/daikin/sensor.py

```diff
@@ -78,51 +79,63 @@ class DaikinSensorEntityDescription(SensorEntityDescription, DaikinRequiredKeysM
     ),
     DaikinSensorEntityDescription(
         key=ATTR_TARGET_HUMIDITY,
-        name="Target Humidity",
+        name="Target humidity",
         device_class=SensorDeviceClass.HUMIDITY,
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PERCENTAGE,
         value_func=lambda device: device.humidity,
     ),
     DaikinSensorEntityDescription(
-        key=ATTR_TOTAL_POWER,
-        name="Estimated Power Consumption",
+        key=ATTR_ALL_POWER,
```

Comment:
Should we consider this a new measurement and the old measurement has been removed? I'm asking since the key has changed which means the unique_id will change and user customization of the old sensor will be lost. Ie, does the new key represent a different kind of measurement? If not, we need to migrate the entity in the entity registry.

Reply:
The measured value has not changed at all, the change of id is to clarify that this sensor measures "all" device's consumption, rather than the "total" consumption of each device. The title has also been changed, and I think it's actually enough to clarify things, so to avoid history data and configuration loss I can revert the change of id

Reply:
Is it okay for you?

Reply:
Sounds good!